src/RequestParser.cpp

```diff
@@ -445,6 +469,69 @@ std::string	RequestParser::parseVersion(const std::string& requestLine)
 	return (requestLine.substr(++i));
 }
 
+
+void	RequestParser::parseChunkedBody(std::istringstream& requestStream)
+{
+	int			length = 0;
+	std::string	strChunkSize;
+	std::getline(requestStream, strChunkSize);
+	if (strChunkSize[strChunkSize.size() - 1] == '\r')
```

Comment:
It's only available in C++20

Reply:
D`oh!

Well to my defense it has been available since C++11, and is constexpr since C++20 :D

If I understand correctly it was added to the 98 standard via DefectReport but not adopted by GNU libstdc++.
Compare with std::map::at() which was also later added and did get adopted. 

Can be resolved after enough bemusement did occur!

Reply:
Where does it say about the Defect Report?

Reply:
https://en.cppreference.com/w/cpp/string/basic_string/back in the last section Defect Reports:

_[...] The following behavior-changing defect reports were applied retroactively to previously published C++ 
[...] C++98: std::basic_string did not have the member function back() | added_

Reply:
Oh I see!